bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -216,8 +222,8 @@ func (gcp *Gcp) GetK8sConfig() (*rest.Config, *clientcmdapi.Config) {
 	// TODO(jlewi): Should we fix this so we can build a cluster config which takes
 	// a TokenSource which can then be pointed at either the DefaultTokenSource
 	// or the refreshable token source?
-	restConfig, err := utils.BuildClusterConfig(ctx, accessToken, gcp.kfDef.Spec.Project,
-		gcp.kfDef.Spec.Zone, gcp.kfDef.Name)
+	restConfig, err := utils.BuildClusterConfig(ctx, accessToken, gcp.kfDef.Project,
```

Comment:
Why isn't it gcp.KfDef.Spec.Project? The whole point of KfConfig is that it should be a drop in replacement.

Reply:
done